app/agent/rest/v1/browse.js

```diff
@@ -0,0 +1,22 @@
+angular.module('portainer.agent')
+.factory('BrowseVersion1', ['$resource', 'API_ENDPOINT_ENDPOINTS', 'EndpointProvider', function BrowseFactory($resource, API_ENDPOINT_ENDPOINTS, EndpointProvider) {
+  'use strict';
+  return $resource(API_ENDPOINT_ENDPOINTS + '/:endpointId/docker/browse/:action', {
```

Comment:
I think that we broke the compatibility with v1 agent in a recent commit, see https://github.com/portainer/portainer/blob/1.19.2/app/agent/rest/browse.js#L4

We need `/docker/browse/:id/:action` as we need to support the volume identifier as a route variable. The `volumeBrowserService` will probably require an update too.

Reply:
I will fix it